_topic_maps/_topic_map.yml

```diff
@@ -2518,6 +2518,8 @@ Topics:
   File: cluster-logging
 - Name: Installing Logging
   File: cluster-logging-deploying
+- Name: Updating Logging
```

Comment:
Is "updating" intentional or should it be "upgrading"

Reply:
The section was always named "updating" (see https://docs.openshift.com/container-platform/4.13/logging/cluster-logging-upgrading.html). I would say since we refer to "updates" throughout the docs (e.g. in release notes https://docs.openshift.com/container-platform/4.13/logging/logging_release_notes/logging-5-7-release-notes.html) it probably makes sense to keep it as updating.